README.md

```diff
@@ -1,149 +1,194 @@
 # sgrep
 
-[![r2c community slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=for-the-badge&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
+[![CircleCI](https://circleci.com/gh/returntocorp/sgrep.svg?style=svg)](https://circleci.com/gh/returntocorp/sgrep)
+[![r2c Community Slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=flat&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
 
-[sgrep.live](https://sgrep.live/): try it now!
+`sgrep` is a tool for easily detecting and preventing bugs and anti-patterns in
```

Comment:
New tagline. Is this first sentence a concise explanation we can get behind and start using elsewhere? E.g. slide decks, Github repository description, external emails, etc.

Reply:
Would like to workshop the tagline more but not a blocker for the PR. I would like to feature sgrep.live up front and probably in even bigger text though -- it's an awesome demo.

Reply:
Roger that, I'm happy to iterate on it now or in the future. My only focus here would be keeping it concise, dead simple, and straightforward so that no one is confused about what the tool actually does.